### DISCUSSION TITLE
Your CF Event Sourcing Video

### DISCUSSION THREAD

**Discussion by fernando-barroso** [mikecann/clouds-and-edges#1] [General]
Hi Michael,

I recently ran into your YouTube video about running Event Sourcing on top of Cloudflare Durable Objects.

https://www.youtube.com/watch?v=W7PmZmVmsDo

I was wondering if we could have a chat? I am building an automation platform and we are planning on using event sourcing as a core pattern. We are really interested in deploying this to Cloudflare. Would love to get your take on your learnings and how you would create something like this now that CF has new products out.

Thank you in advance!

**Comment by mikecann** [comment]
Hi Fernando,

Oh fantastic, its great you found some value from my little project.Its
been a quite a while since I worked on this and I haven't been keeping 100%
up to date with all the latest goings on but I would totally be up for
having a chat about your ideas and I will help out any way I can.

Shoot me an email: ***@***.*** and we can sort out a time for a
call.

Cheers,

*Mike Cann*
http://www.mikecann.co.uk/
@mikeysee <http://www.twitter.com/mikeysee>

**Reply by fernando-barroso** [reply]
Hey Mike! Thanks for the quick response. I am not able to see your email unfortunately (looks like GitHub hides those automatically). I pinged you via Twitter, but if you'd like here's my calendar link: https://cal.com/fernandobarroso/30min

Any chance you have some time to link up tomorrow or next week? Thanks in advance!

**Comment by mikecann** [comment]
Hi Fernando,

I forgot to reply here but I booked the call and am on the meeting now..

*Mike Cann*
http://www.mikecann.co.uk/
@mikeysee <http://www.twitter.com/mikeysee>
